### PR TITLE
Aldata

### PR DESCRIPTION
**Pull request by elliottabirch**

### CODE REVIEW

File: src/util/reports/campPhotos.js [hunk 1]
```diff
@@ -60,9 +60,15 @@ module.exports = (startDate, endDate) =>
     .flatMap(sessions => assignDocumentData(sessions, streamRegistrations, 'sessionId', 'sessionId', 'registration'))
     .flatMap(({ registration, ...rest }) => hl(registration.registrationDetails || [])
       .filter(({ cancelled }) => !cancelled)
-      .map(({ tuitionId, personId, sessionOptions }) => Object.assign({ tuitionId, personId, sessionOptions }, rest)))
-    .collect()
+	  //.map(({ tuitionId, personId, sessionOptions }) => Object.assign({ tuitionId, personId, sessionOptions }, rest)))
```
Comment (elliottabirch) [inline src/util/reports/campPhotos.js]: go ahead adn remove these commented out lines

File: src/util/reports/util.js [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
+} = require("./constants");
+
 
+const createUrl = (url, endPoint) => `${url}/${endPoint}`;
 const parseBuffer = stream => stream
   .collect()
   .map(buffers => buffers.join(''))
   .map(res => JSON.parse(res));
-
+/*
```
Comment (elliottabirch) [inline src/util/reports/util.js:22]: these commetns can be removed

File: src/util/reports/campPhotos.js [hunk 3]
```diff
@@ -60,9 +60,7 @@ module.exports = (startDate, endDate) =>
     .flatMap(sessions => assignDocumentData(sessions, streamRegistrations, 'sessionId', 'sessionId', 'registration'))
     .flatMap(({ registration, ...rest }) => hl(registration.registrationDetails || [])
       .filter(({ cancelled }) => !cancelled)
-      .map(({ tuitionId, personId, sessionOptions }) => Object.assign({ tuitionId, personId, sessionOptions }, rest)))
-    .collect()
-    .flatMap(assignHasPurchasedPhotos)
```
Comment (elliottabirch) [inline src/util/reports/campPhotos.js:65]: why was this removed?

Comment (pankajr03) [reply src/util/reports/campPhotos.js:65]: SessionOptions are not be used for any field for excel creation, so i think it will be an extra request, so i remove it.
